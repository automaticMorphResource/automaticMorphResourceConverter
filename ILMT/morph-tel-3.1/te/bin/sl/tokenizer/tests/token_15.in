అందుకనే ఆస్ట్రేలియా ప్రభుత్వం క్వీన్స్ లాండ్ ప్రభుత్వ సహకారంతో ఈ ప్రాంతాన్నంతటినీ పరిరక్షిత ప్రాంతంగా ప్రకటించి అనేక రక్షణ చర్యలను చేపడుతోంది. 
