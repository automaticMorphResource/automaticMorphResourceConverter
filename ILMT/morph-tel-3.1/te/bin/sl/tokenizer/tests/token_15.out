<Sentence id="1">
1	అందుకనే	unk
2	ఆస్ట్రేలియా	unk
3	ప్రభుత్వం	unk
4	క్వీన్స్	unk
5	లాండ్	unk
6	ప్రభుత్వ	unk
7	సహకారంతో	unk
8	ఈ	unk
9	ప్రాంతాన్నంతటినీ	unk
10	పరిరక్షిత	unk
11	ప్రాంతంగా	unk
12	ప్రకటించి	unk
13	అనేక	unk
14	రక్షణ	unk
15	చర్యలను	unk
16	చేపడుతోంది	unk
17	.	unk
</Sentence>

